backend/prisma/migrations/20240729210931_init_favourites_table/migration.sql

```diff
@@ -0,0 +1,21 @@
+-- CreateTable
+CREATE TABLE "Favourite" (
+    "id" SERIAL NOT NULL,
+    "UserId" INTEGER NOT NULL,
+    "adopterId" INTEGER,
+    "adopteeId" INTEGER,
```

Comment:
If the user is an adopter and likes the profile of an adoptee,  the adopteeId is stored. Therefore the adoptee id is the number of the adoptee being liked.